yarn/src/test/scala/org/apache/spark/deploy/yarn/ClientSuite.scala

```diff
@@ -43,12 +45,21 @@ import org.apache.spark.util.Utils
 
 class ClientSuite extends SparkFunSuite with Matchers with BeforeAndAfterAll {
 
+  var oldProperties: Properties = null
```

Comment:
That's a fair point -- yes it's called two lines below, right? I think you could use the standard mechanism in the 2 places you're manually copying the sys properties?